mongomock/helpers.py

```diff
@@ -86,6 +92,8 @@ def create_index_list(key_or_list, direction=None):
     """
     if isinstance(key_or_list, string_types):
         return [(key_or_list, direction or ASCENDING)]
+    if isinstance(key_or_list, ItemsView):
+        return list(key_or_list)
     if not isinstance(key_or_list, (list, tuple)):
```

Comment:
Add it at the end here: `if not isinstance(key_or_list, (list, tuple, Iterable):` as we don't need to cast it as a list